src/wrapper.ts

```diff
@@ -0,0 +1,368 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import assert = require('assert');
+import translate = require('./translate');
+import requireFromString = require('require-from-string');
+import https = require('https');
+import MemoryStream = require("memorystream");
+import semver = require('semver');
+import { IncomingMessage } from 'http';
+
+export class Wrapper {
```

Comment:
@axic Can I leave as a class or rather revert it to the setupMethods as it was before?